common.mk

```diff
@@ -1,4 +1,6 @@
 ARCH=$(shell uname -m)
+# if this fails on centos-based OS: yum install -y epel-release && yum install -y dpkg
+DPKG_ARCH=$(shell dpkg --print-architecture)
```

Comment:
Should this go in the deb/Makefile? (I think it's only used there, correct?)

Reply:
it's already in that makefile and in rpm/Makefile as well via an include.

Reply:
Ah sorry misunderstood. It is used in both deb and rpm Makefiles.

Reply:
Looks like this is not super important, as nothing checks the architecture. This is what is consuming it: https://github.com/docker/release-repo/blob/031eeb8a6e4b99ab08b7c12e72f29a8d5dde64da/Makefile#L66

Reply:
I just wanted to keep the same names as in release-packaging https://github.com/docker/release-packaging/blob/357e79197ac898fe1a2a89b3120c16df7de4a07b/Makefile#L108